docker/runtime/backup-init.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+set -e
+
+BACKUP_PATH="/backup"
+
+# Make sure that required directories exist
+mkdir -p "${BACKUP_PATH}"
+mkdir -p "/etc/crontabs"
+chown git:git /backup
+chmod 2770 /backup
+
+# [string] BACKUP_INTERVAL   Period expression
+# [string] BACKUP_RETENTION  Period expression
+if [ -z "${BACKUP_INTERVAL}" ]; then
+	echo "Backup disabled: BACKUP_INTERVAL has not been found" 1>&2
+	exit 1
```

Comment:
If the user has `CROND` enabled but not backup, the whole thing will fail? i.e. should it be `exit 0`?

Reply:
backup-init.sh is started is as a separated process, so it's exit code will not interfere with the main shell process (start.sh). Exit code 1 allows us to check exit code in start.sh. 

If I used `source backup-init.sh` that would be a different story. :smile:

Reply:
Ah cool, TIL, thanks :)